cli_tools_tests/e2e/gce_ovf_import/test_suites/ovf_instance_import/ovf_instance_import_test_suite.go

```diff
@@ -591,6 +622,22 @@ func verifyImportedInstance(
 		}
 	}
 
+	if props.instanceAccessScopes != "" {
```

Comment:
if the scope is not specified, what's the most proper value? Maybe make it align with the result of "gcloud compute instances create"?

Reply:
Added default scopes if not specified by user